### PR TITLE
Config

### PR DESCRIPTION
**Pull request by lkitching**

### CODE REVIEW

File: resources/get-unmapped-dimension-values.sparql [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>
+PREFIX ui: <http://www.w3.org/ns/ui#>
+
+SELECT ?dim ?member ?label WHERE {
+  ?ds qb:structure ?struct .
+  ?struct a qb:DataStructureDefinition .
+  ?struct qb:component ?comp .
+  VALUES ?dim { ?refareadim ?refperioddim }
+  ?comp qb:dimension ?dim .
+  ?comp qb:codeList ?list .
```
Comment (zeginis) [inline resources/get-unmapped-dimension-values.sparql]: The source for the code list can be either the component (?comp) or the dimension (?dim). This information should be red from the configuration `codelist-source`

File: resources/get-unmapped-dimension-values.sparql [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+  ?list skos:member ?member .
+  OPTIONAL { ?member rdfs:label ?label . }
```
Comment (zeginis) [inline resources/get-unmapped-dimension-values.sparql]: The ?member label property can be other that the rdfs:label e.g. skos:prefLabel. This information should be red from the configuration `codelist-label`

**Review by zeginis — CHANGES_REQUESTED** [review]
@lkitching thank you for the PR. It looks very good. I have only two comments for the get-unmapped-dimension-values.sparql
- the source of the codelist could be either the ?comp or the ?dim. This should be red from the configuration
- the member label can be other than the rdfs:label. This should be red from the configuration

**Comment by lkitching** [comment]
@zeginis - Thanks, I didn't notice those during the refactor; I've reverted that commit.

**Pull request merged**